f2/src/forms/ContactInfoFormSchema.js

```diff
@@ -57,26 +47,58 @@ const createErrorSummary = (errors) => {
     }
   }
 
-  if (email) {
-    errorSummary['email'] = {
-      label: contactFormOptions.email.label,
-      message: contactFormOptions.email.errorMessage,
+  if (emailORphone) {
+    errorSummary['emailORphone'] = {
+      label: contactFormOptions.emailORphone.label,
+      message: contactFormOptions.emailORphone.errorMessage,
     }
   }
 
-  if (phone) {
-    errorSummary['phone'] = {
-      label: contactFormOptions.phone.label,
-      message: contactFormOptions.phone.errorMessage,
-    }
+  return errorSummary
+}
+
+const fyiValidate = (values) => {
+  const errors = {}
+  const fields = {}
+
+  if (values.email && !addrs(values.email)) {
```

Comment:
I don't think the email-addresses module is viable for email address validation. We should use Yup since we already have valid email defined there. You can just call the validate function manually for emails instead of passing the whole thing to Formik.

As the validation stands now invalid emails will be accepted, such as justin.email@a

Reply:
Solved, new fine-tuned TUP validation is in place